src/masoniteorm/relationships/BaseRelationship.py

```diff
@@ -52,7 +52,7 @@ def __get__(self, instance, owner):
             object -- Either returns a builder or a hydrated model.
         """
         attribute = self.fn.__name__
-        relationship = self.fn(self)()
+        relationship = self.fn(instance)()
```

Comment:
should this be the intance variable or the owner variable? i think owner. instance would be the class but owner would be the initialized object.

Reply:
`owner` appears to be the class, `User` in my example, whereas `instance` is an instance of `owner`. `self.fn` is a pointer to an instance method (Ruby vernacular) or prototype function (Node) -- apologies, I don't know the corresponding term in Python -- so I _believe_ `instance` is correct